src/Io/ServerRequest.php

```diff
@@ -134,17 +141,16 @@ public function withoutAttribute($name)
     }
 
     /**
-     * @internal
      * @param string $cookie
-     * @return boolean|mixed[]
+     * @return array()
```

Comment:
```suggestion
     * @return array
```

Reply:
Duh, thanks for spotting, PR updated! :shipit: